db/seeder/mysql.config.js.template

```diff
@@ -0,0 +1,8 @@
+const mysqlConfig = {
+  host: 'localhost',
+  user: '[MYSQL USER]',
+  password: '[MYSQL PASS]',
+  database: 'gallery',
+};
+
+module.exports = mysqlConfig;
```

Comment:
See this little red symbol at the end of the line here? I have strong feelings about this. Here's an explanation why git is warning you about this file not ending in a newline https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725#729725